runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -235,17 +235,12 @@ void registerTable(@Nonnull String namespace,
              * Caller is opening a new table with a map of protobufFilename -> protobufFileDescriptor pairs
              * Here are the cases that can happen:
              * 1. No entry in TableRegistry -> New table
-             *  1.1: When inserting new protobufs into descriptor table, we find existing entries and they all match
-             *  1.2: When inserting new protobufs into descriptor table, we find mismatching [Schema Change!]
+             *  1.1: When inserting new protobufs into descriptor table, we find existing entries, we don't replace
```

Comment:
Can you rewrite this whole comment. There is no validation here (old and new code): it doesn't reflect the code.

Reply:
done!